BlazarUI/app/scripts/components/shared/PageHeader.jsx

```diff
@@ -1,6 +1,19 @@
 import React, {Component, PropTypes} from 'react';
+import $ from 'jquery';
```

Comment:
@benrlodge The react animation docs seem to only cover the use case of animating components as the enter or leave the DOM. Does it work if we want to modify an existing element?